web/src/Routes.jsx

```diff
@@ -16,23 +16,25 @@ import { useAuth } from './auth'
 const Routes = () => {
   return (
     <Router useAuth={useAuth}>
-      <Set wrap={ScaffoldLayout} title="Users" titleTo="users" buttonLabel="New User" buttonTo="newUser">
-        <Route path="/users/new" page={UserNewUserPage} name="newUser" />
-        <Route path="/users/{id}/edit" page={UserEditUserPage} name="editUser" />
-        <Route path="/users/{id}" page={UserUserPage} name="user" />
-        <Route path="/users" page={UserUsersPage} name="users" />
-      </Set>
-      <Set wrap={ScaffoldLayout} title="Classrooms" titleTo="classrooms" buttonLabel="New Classroom" buttonTo="newClassroom">
-        <Route path="/classrooms/new" page={ClassroomNewClassroomPage} name="newClassroom" />
-        <Route path="/classrooms/{id:Int}/edit" page={ClassroomEditClassroomPage} name="editClassroom" />
-        <Route path="/classrooms/{id:Int}" page={ClassroomClassroomPage} name="classroom" />
-        <Route path="/classrooms" page={ClassroomClassroomsPage} name="classrooms" />
-      </Set>
+      <PrivateSet unauthenticated="login">
+        <Set wrap={ScaffoldLayout} title="Users" titleTo="users" buttonLabel="New User" buttonTo="newUser">
```

Comment:
Really good fix, just commit with `fix:` tag, like `fix: routes bug..` and describe